joblib/memory.py

```diff
@@ -130,9 +136,72 @@ def _load_output(output_dir, func_name, timestamp=None, metadata=None,
         raise KeyError(
             "Non-existing cache value (may have been cleared).\n"
             "File %s does not exist" % filename)
-    return numpy_pickle.load(filename, mmap_mode=mmap_mode)
+    result = numpy_pickle.load(filename, mmap_mode=mmap_mode)
+
+    return result
+
+
+def _get_cache_items(root_path):
+    """Get cache information for reducing the size of the cache"""
+    cache_items = []
+
+    for dirpath, dirnames, filenames in os.walk(root_path):
+        is_cache_hash_dir = re.match('[a-z0-9]{32}', os.path.basename(dirpath))
```

Comment:
I think the regexp should be `[a-f0-9]{32}` (replace `z` with `f`) as it's a string containing hexadecimal characters.